test/profile/test-storage.js

```diff
@@ -16,7 +16,7 @@ import { ProfileStorage, VisitType } from '../../app/services/storage';
 
 describe('Utility tests', () => {
   it('Compares sets correctly.', (done) => {
-    expect(Immutable.Set(['a', 'b']).equals(Immutable.Set(['b', 'a'])));
+    expect(Immutable.Set(['a', 'b']).equals(Immutable.Set(['b', 'a']))).toBe(true);
```

Comment:
Is there a different between `toBe` and `toEqual`?  Is one `==` and the other `===`?